recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -134,9 +143,10 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.rmdir(os.path.join(self.package_folder, "doc"))
         # remove binaries and pdb files
-        for pattern_to_remove in ["cjpeg*", "djpeg*", "jpegtran*", "tjbench*", "wrjpgcom*", "rdjpgcom*", "*.pdb"]:
-            for bin_file in glob.glob(os.path.join(self.package_folder, "bin", pattern_to_remove)):
-                os.remove(bin_file)
+        if not self.options.keep_binaries:
+            for pattern_to_remove in ["cjpeg*", "djpeg*", "jpegtran*", "tjbench*", "wrjpgcom*", "rdjpgcom*", "*.pdb"]:
+                for bin_file in glob.glob(os.path.join(self.package_folder, "bin", pattern_to_remove)):
+                    os.remove(bin_file)
```

Comment:
don't keep pdb files.

Reply:
yes, this is untreated change

Reply:
If the executables are already generated, why removing them from the package? I would say that storage is not an issue nowadays (at least, it isn't for us in ConanCenter). I would remove `keep_binaries` option and keep these binaries in the package.

Reply:
@KristianJerpetjon 
We have an issue for discussing including pdbs: https://github.com/conan-io/conan-center-index/issues/1982 
https://github.com/conan-io/conan-center-index/issues/1982#issuecomment-659455118 seems to suggest a conan feature will be added for handling pdb's.